### PR TITLE
Improve VDI documentation

### PR DESCRIPTION
**Pull request by jobarner**
Clarify VDI scenarios, following a customer case which wasn't clear about machine names

### CODE REVIEW

**Comment by illfated** [comment]
Suggest that you replace "improvate" with simply "improve", because "improvate" is a modern business word built from the combination of improve + innovate.

**Improve VDI documentation**

**Comment by jobarner** [comment]
> Suggest that you replace "improvate" with simply "improve", because "improvate" is a modern business word built from the combination of improve + innovate.
> 
> **Improve VDI documentation**

Changed, that was a typo

File: windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-vdi.md [hunk 1]
```diff
@@ -31,17 +31,19 @@ ms.date: 04/24/2018
 
 Microsoft Defender ATP supports non-persistent VDI session onboarding. There might be associated challenges when onboarding VDIs. The following are typical challenges for this scenario:
 
+- Instant early onboarding of a short-lived sessions, which must be onboarded to Microsoft Defender ATP prior to the actual provisioning.
+- Machine name are typically reused for new sessions.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-vdi.md]:
```suggestion
- Machine names are typically reused for new sessions.
```
(alternatively, the other way around: "`The machine name is typically reused for new sessions.`")

Comment (jobarner) [reply windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-vdi.md]: Done

**Comment by illfated** [comment]
@e0i : Does it fall under your "jurisdiction" to follow up on this PR?

**Comment by jobarner** [comment]
> @e0i : Does it fall under your "jurisdiction" to follow up on this PR?
@mjcaparas Usually handles documentation for `microsoft-defender-atp`

**Comment by illfated** [comment]
> > @e0i : Does it fall under your "jurisdiction" to follow up on this PR?

> @mjcaparas Usually handles documentation for `microsoft-defender-atp`

Sure, @mjcaparas is the document author, the same way as @Dansimp is the doc manager. I meant to ask for one of the case handlers (@e0i / @mypil / @jvsam ) who usually make sure to notify those who are needed to follow up with the next step in issue ticket handling and PR handling.

**Pull request merged**